README.md

```diff
@@ -70,3 +83,4 @@ npm test
 | [detect-unsafe-regex](docs/rules/detect-unsafe-regex.md)                                     | Detects potentially unsafe regular expressions, which may take a very long time to run, blocking the event loop.              | ✅  |
 
 <!-- end auto-generated rules list -->
+````
```

Comment:
Is this a mistake?

Reply:
ops, it was auto-gened, will take a look.

Reply:
There was an issue with the formatting of the js code block above, which should be fine now.